### DISCUSSION TITLE
Vaultwarden Secret Manager

### DISCUSSION THREAD

**Discussion by Y0ngg4n** [dani-garcia/vaultwarden#3924] [Ideas]
Would be nice to have something like that https://bitwarden.com/products/secrets-manager/ with vaultwarden infrastructure. 🙂

**Comment by BlackDex** [comment]
If you would have searched, you would have found that will not happen.

**Reply by Y0ngg4n** [reply]
@BlackDex Sorry did not found a discussion to that. Then we can close this Discussion.

**Reply by stefan0xC** [reply]
See https://github.com/dani-garcia/vaultwarden/discussions/3368 or https://github.com/dani-garcia/vaultwarden/discussions/3278 (and also https://github.com/dani-garcia/vaultwarden/issues/3793)
